include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10838,6 +10838,18 @@
       "capabilities" : [ "TernaryBitwiseFunctionINTEL" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpUntypedVariableLengthArrayINTEL",
+      "class"  : "@exclude",
```

Comment:
This is consistent with the rest of the variable length array instructions, but I think we should remove `@exclude` now and add all of the variable length array instructions to some other class instead (maybe Memory?).  Rationale: There aren't very many variable length array instructions, and the extension will be published on the registry shortly.

Reply:
+1, I don't think exclude is right (unless you really want to hide this in the core spec). Memory is more appropriate (and that's what the extension mentions anyway).

Reply:
Applied, thanks!